### ISSUE TITLE
preload_basic_commands test fails when executed from root

### ISSUE THREAD

**Issue by marcinslusarz** [pmem/pmemfile#289]
It takes write permission from a file and expects overwriting it should fail.

**Comment by GBuella** [comment]
Those tests should not be run as root.

**Comment by marcinslusarz** [comment]
I agree. We just need to detect it and refuse to run tests at all (I think it's preferred) or skip some of them.

**Comment by GBuella** [comment]
Yup, libpmemfile should just abort when root.

**Comment by marcinslusarz** [comment]
libpmemfile tests, not libpmemfile!

**Comment by GBuella** [comment]
Well, both, that is the best.

**Comment by marcinslusarz** [comment]
Then we wouldn't be able to test libpmemfile with pjdfstest. And I don't see any problem with handling root correctly. We already support chown, capabilities affecting file APIs, setuid & co.
